_includes/card-problem-messages.md

```diff
@@ -5,6 +5,20 @@ problem message. We generally use the problem message `type` and `status` code
 to identify the nature of the problem. The problem `name` and `description` will
 often help narrow down the specifics of the problem.
 
+### Common Problem Types
```

Comment:
These are common across all payment instruments. Perhaps we can create an include for it to reuse everywhere?